packages/openneuro-app/src/scripts/datalad/dashboard/datasets/dataset-sorter.jsx

```diff
@@ -50,22 +51,50 @@ SortField.propTypes = {
   refetch: PropTypes.func,
 }
 
-const DatasetSorter = ({ queryVariables, refetch }) => (
-  <>
-    {sortFields.map(field => (
-      <SortField
-        field={field}
-        queryVariables={queryVariables}
-        refetch={refetch}
-        key={field}
-      />
-    ))}
-  </>
-)
+const DatasetSorter = ({ queryVariables, refetch }) => {
+  const isMobile = useMedia('(max-width: 700px) ')
+  const onChange = event => {
+    const newQueryVariables = { ...queryVariables }
+    newQueryVariables.orderBy = {}
+    newQueryVariables.orderBy[event.target.value] =
+      queryVariables.orderBy[event.target.value] === 'descending'
```

Comment:
This will always evaluate to false because the event only fires on change. Reversing the sort order will need some additional button/options or some fancy logic (design Q).

Reply:
Just adding a note for documentation: the now revised solution entails being able to include ascending / descending sort on mobile. Originally, the toggling caret wasn't going to work for a dropdown, but given we will imminently be consolidating / removing some of the sort options, we can add both asc/desc options for each sort item, respectively.